src/main/java/com/minecolonies/coremod/placementhandlers/main/SurvivalHandler.java

```diff
@@ -92,6 +92,13 @@ public void handle(
       final BlockPos blockPos,
       final PlacementSettings placementSettings)
     {
+        if (blueprint == null)
+        {
+            // This can happen if the file didnt finish synching with the server from the client, or something went wrong when synching (package dropped, etc).
+            MessageUtils.format(NO_CUSTOM_BUILDINGS).sendTo(player);
```

Comment:
Shouldn't this be `BUILDING_MISSING` instead?

Reply:
derp